packages/cli/config/index.ts

```diff
@@ -639,6 +639,45 @@ const config = convict({
 		},
 	},
 
+	telemetry: {
+		enabled: {
+			doc: 'Whether telemetry is enabled.',
+			format: Boolean,
+			default: true,
+			env: 'N8N_TELEMETRY_ENABLED',
+		},
+		config: {
+			frontend: {
+				key: {
+					doc: 'Rudder key for frontend.',
+					format: String,
+					default: '1wfyHJrZGsJmqh86XcncyX2SN3T',// todo update KEY
+					env: 'N8N_TELEMETRY_RUDDER_KEY_FRONTEND',
```

Comment:
We could remove `RUDDER` from all the env vars names as we might change the service in the future.